package.json

```diff
@@ -20,12 +20,12 @@
     "ember-auto-focus": "1.0.9",
     "ember-autoresize": "0.6.0",
     "ember-can": "^0.8.4",
-    "ember-cli": "2.13.1",
+    "ember-cli": "~2.14.0-beta.2",
     "ember-cli-app-version": "^3.0.0",
     "ember-cli-autoprefixer": "0.7.0",
-    "ember-cli-babel": "^5.2.4",
+    "ember-cli-babel": "^6.4.1",
     "ember-cli-bourbon": "2.0.0-beta.1",
-    "ember-cli-code-coverage": "0.3.12",
+    "ember-cli-code-coverage": "kategengler/ember-cli-code-coverage",
```

Comment:
- 0.3.12 fails with the new ember-cli version
- 0.3.20 resolves that issue, but there's a new one, related to babel
- The babel issue is fixed in master, but a new version has not been released yet.